jms/src/main/scala/akka/stream/alpakka/jms/JmsConnector.scala

```diff
@@ -49,49 +44,111 @@ private[jms] trait JmsConnector { this: GraphStageLogic =>
       case d => d
     }
 
-    materializer match {
-      case m: ActorMaterializer => m.system.dispatchers.lookup(dispatcher.dispatcher)
-      case x => throw new IllegalArgumentException(s"Stage only works with the ActorMaterializer, was: $x")
-    }
+    ActorMaterializerHelper.downcast(materializer).system.dispatchers.lookup(dispatcher.dispatcher)
```

Comment:
Replaced with Akka library call providing the same functionality.

Reply:
👍